extension/src/popup/views/ExperimentalFeatures/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useSelector, useDispatch } from "react-redux";
+import { Notification, Button, Toggle } from "@stellar/design-system";
+import { Field, Form, Formik } from "formik";
+
+import {
+  saveExperimentalFeatures,
+  settingsSelector,
+} from "popup/ducks/settings";
+
+import { SubviewHeader } from "popup/components/SubviewHeader";
+import { AutoSaveFields } from "popup/components/AutoSave";
+import { View } from "popup/basics/layout/View";
+import IconExperimental from "popup/assets/icon-settings-experimental.svg";
+
+import "./styles.scss";
+
+export const ExperimentalFeatures = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [isUnderstood, setIsUnderstood] = useState(false);
+
+  const { isExperimentalModeEnabled, isHashSigningEnabled } =
+    useSelector(settingsSelector);
+
+  interface SettingValues {
+    isExperimentalModeEnabledValue: boolean;
+    isHashSigningEnabledValue: boolean;
+  }
+
+  const initialValues: SettingValues = {
+    isExperimentalModeEnabledValue: isExperimentalModeEnabled,
+    isHashSigningEnabledValue: isHashSigningEnabled,
+  };
+
+  const handleSubmit = async (formValue: SettingValues) => {
```

Comment:
Does this action need a loading state?

Reply:
I added this, though it just shows for a split second so may be hard to see

<img width="762" alt="Screenshot 2024-05-28 at 4 22 13 PM" src="https://github.com/stellar/freighter/assets/6789586/b91008d6-03d5-4f0b-952a-7f2d59c1a3d5">

Reply:
Thank yeah I did honestly mean that as a question cause if it doesn't make a network trip, even though it's an action, sometimes it can be janky with the very fast loading state but I don't think we have a real pattern for when it is or isn't appropriate I think.